src/targeting/personalised.ts

```diff
@@ -0,0 +1,256 @@
+import type { ConsentState } from '@guardian/consent-management-platform/dist/types';
+import type {
+	TCEventStatusCode,
+	TCFv2ConsentList,
+} from '@guardian/consent-management-platform/dist/types/tcfv2';
+import { storage } from '@guardian/libs';
+import { clearPermutiveSegments, getPermutiveSegments } from '../permutive';
+import type { False, NotApplicable, True } from '../types';
+
+/* -- Types -- */
+
+const frequency = [
+	'0',
+	'1',
+	'2',
+	'3',
+	'4',
+	'5',
+	'6-9',
+	'10-15',
+	'16-19',
+	'20-29',
+	'30plus',
+	'5plus', // TODO: remove it (?)
```

Comment:
Is the `5plus` a value that's still in GAM? Can we remove it from here if `getFrequencyValue` never returns a value of `5plus`?

Reply:
https://github.com/guardian/frontend/pull/11297 I can confirm it’s been deprecated since Dec 2015